psl-core/src/main/java/org/linqs/psl/application/learning/weight/search/grid/GridSearch.java

```diff
@@ -57,9 +51,11 @@ public GridSearch(List<Rule> rules, Database rvDB, Database observedDB) {
         }
 
         maxNumLocations = (int)Math.pow(possibleWeights.length, mutableRules.size());
+
         numLocations = maxNumLocations;
     }
 
+
```

Comment:
Extra newline.
You seem to add in this extra newline (between methods) a lot, maybe your editor?